src/Api/MediaLibraryApi.php

```diff
@@ -233,25 +219,29 @@ public function mediaFilesGet(?int $limit = 20, ?int $offset = 0, ?string $flavo
         {
           continue;
         }
-        $mediaFile = [
-          'id' => $media_package_file->getId(),
-          'name' => $media_package_file->getName(),
-          'flavor' => $media_package_file->getFlavor(),
-          'package' => $media_package_file->getCategory()->getPackage()->first()->getName(),
-          'category' => $media_package_file->getCategory()->getName(),
-          'author' => $media_package_file->getAuthor(),
-          'extension' => $media_package_file->getExtension(),
-          'download_url' => $this->url_generator->generate(
-              'download_media',
-              ['id' => $media_package_file->getId()],
-              UrlGenerator::ABSOLUTE_URL),
-        ];
 
-        $json_response_array[] = new MediaFile($mediaFile);
+        $json_response_array[] = new MediaFile($this->addMediaFile($media_package_file));
       }
     }
     $responseData = new MediaFiles(['media_files' => $json_response_array, 'total_results' => $total_results]);
 
     return $responseData;
   }
+
+  public function addMediaFile(MediaPackageFile $media_package_file)
```

Comment:
Since this method is not adding a new MediaFile, it would be better to rename it so something like `getMediaFileResponseData`